src/main/java/nl/hsac/fitnesse/util/RandomPostalCodeGenerator.java

```diff
@@ -0,0 +1,80 @@
+package nl.hsac.fitnesse.util;
+
+import java.util.Random;
+
+public class RandomPostalCodeGenerator {
+    RandomUtil random = new RandomUtil();
```

Comment:
Is there a reason this field cannot be `private static final`?

Reply:
no, so I changed it.